scss/_navbar.scss

```diff
@@ -204,19 +204,20 @@
     }
   }
 
-  .navbar-nav {
-    .nav-link {
-      color: $navbar-light-color;
+  .nav-link,
+  .btn-link {
+    color: $navbar-light-color;
 
-      @include hover-focus {
-        color: $navbar-light-hover-color;
-      }
+    @include hover-focus {
+      color: $navbar-light-hover-color;
+    }
 
-      &.disabled {
-        color: $navbar-light-disabled-color;
-      }
+    &.disabled {
```

Comment:
If we're adding a `.btn-` class, we'll likely need `:disabled` in these selectors as well.